opencti-platform/opencti-graphql/src/utils/emailTemplates/footer.ts

```diff
@@ -0,0 +1,39 @@
+export const FOOTER_TEMPLATE = `
+                        <tr>
+                           <td>
+                              <table id="footer" border="0" cellpadding="0" cellspacing="0" height="100%" width="100%" bgcolor="#f5f8fa" style="width: 100%; min-width: 100%;">
```

Comment:
maybe just extract the table ? It's more readable to me when there is a complete chunk of html and not a part in one constant and another part in one other.